mlir/include/mlir/Dialect/Polynomial/IR/Polynomial.h

```diff
@@ -0,0 +1,129 @@
+//===- Polynomial.h - A storage class for polynomial types ------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_INCLUDE_MLIR_DIALECT_POLYNOMIAL_IR_POLYNOMIAL_H_
+#define MLIR_INCLUDE_MLIR_DIALECT_POLYNOMIAL_IR_POLYNOMIAL_H_
```

Comment:
Nit: let's not have the `MLIR_INCLUDE` part, the rest of the codebase starts with `MLIR_DIALECT`.

Reply:
Fixed in 5154bee21d4